bears/general/CPDBear.py

```diff
@@ -102,6 +109,8 @@ def run(self, language: str, minimum_tokens: int=20,
 
         stdout_output, _ = run_shell_command(arguments)
 
+        os.remove(tmp_filename)
```

Comment:
Can we have a test for this? Feed the same file a 100,000 times.

Reply:
While useful, I dont think it is worth the effort.